ui-particles-angular/projects/ui-particles-angular/src/lib/gio-save-bar/gio-save-bar.component.scss

```diff
@@ -22,7 +22,7 @@ $typography: map.get(gio.$mat-theme, typography);
 
 :host {
   display: block;
-  height: 50px;
+  height: 68px;
```

Comment:
Be careful, with this and integration in console api design

Reply:
👌🏻  Yes, and we need to check with cockpit and @audrey-tlm first